PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -1621,15 +1617,15 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 
 		case NPPM_INTERNAL_ENABLECHECKDOCOPT:
 		{
-			NppGUI & nppgui = const_cast<NppGUI &>((pNppParam->getNppGUI()));
+			NppGUI& nppgui = const_cast<NppGUI&>((pNppParam->getNppGUI()));

```

Comment:
@donho 
**Additional note:**
I feel "Doc Monitor" plugin developer needs to intimated. With this current implementation, new functionalities can not be enabled. So this plugin developer should be informed and we may need to provide additional support to him/her if s/he is actively updating plugin.